api/express.ts

```diff
@@ -50,12 +50,15 @@ const corsOptions = {
   credentials: true,
 }
 
+const maxBodySize = '10mb';
+
 // Incoming CORS Filter
 app.use(cors(corsOptions));
 
 // Express middleware
-app.use(express.json());
-app.use(express.urlencoded({ extended: false }));
+app.use(express.json({ limit: maxBodySize }));
+app.use(express.urlencoded({ extended: false, limit: maxBodySize }));
+app.use(express.text({ limit: maxBodySize }))
```

Comment:
What does this do? Docs says it parses incoming requests into text but you also have express.json that parses bodies.

Reply:
The default request size limit is only 100KB. I'm trying to start the file limit at 10MB, but probably I can even dial it down from there. If I don't specify, then the request gets denied for exceeding the size limit. 

If you mean the .text(), I'm thinking that might have been leftover from a previous attempt with multi-part form data...
Seems to work without it, so I'm going to remove it.